### ISSUE TITLE
Error occurs when calculating expection of a constant polynomial

### ISSUE THREAD

**Issue by cc636489** [jonathf/chaospy#230]
I tried to calculate the expectation of a constant polynomial, which is a corner case I'd like to merge into common cases. 

==== Re-produce the error in chaospy-3.2.9:

```python
from chaospy import *
dist = J(Uniform(-1.2, 1.2), Uniform(-2.0, 2.0))
a = poly.polynomial(1.0)
E(a, dist)
```
==== Expect to return a value 1.0.

==== Error occurs stated as below:

Traceback (most recent call last):
  File ".../chaospy/descriptives/expected.py", line 38, in E
    poly = polynomials.setdim(poly, len(dist))
  File ".../chaospy/poly/setdim.py", line 30, in setdim
    indices = [int(name[1:]) for name in poly.names]
ValueError: invalid literal for int() with base 10: ''

Am I supposed to consider this case separately? 
Thanks!

**Comment by jonathf** [comment]
No, it is a bug. Working on a fix.

**Comment by jonathf** [comment]
Should be fixed in v3.2.10. Let me know.
